source/wp-content/themes/wporg-developer-2023/js/user-notes.js

```diff
@@ -3,73 +3,79 @@
  *
  */
 
-( function( $ ) {
+( function () {
+	const commentForm = document.querySelector( '.comment-form textarea' );
+	let commentID = window.location.hash;
+	let wpAdminBar = 0;
 
-	var commentForm = $( '.comment-form textarea' );
-	var commentID = window.location.hash;
-	var wpAdminBar = 0;
-
-	// Check if the fragment identifier is a comment ID (e.g. #comment-63)
+	// Check if the fragment identifier is a comment ID (e.g. #comment-63).
 	if ( ! commentID.match( /#comment\-[0-9]+$/ ) ) {
 		commentID = '';
 	}
 
-	// Actions for when the page is ready
-	$( document ).ready( function() {
-		// Set wpAdminBar
-		wpAdminBar = $( '#wpadminbar' ).length ? 32 : 0;
-
+	// Actions for when the page is ready.
+	document.addEventListener( 'DOMContentLoaded', function () {
+		// Set wpAdminBar.
+		wpAdminBar = document.querySelector( '#wpadminbar' ).length ? 32 : 0;
 		// Display form and scroll to it
 		if ( '#respond' === window.location.hash ) {
 			showCommentForm();
 		}
-
-		if( ! wpAdminBar || ! commentID ) {
+		if ( ! wpAdminBar || ! commentID ) {
 			return;
 		}
-
-		var comment = $('#comments').find( commentID + '.depth-1' ).first();
-		if( ! comment.length  ) {
+		const comment = document
+			.querySelector( '#comments' )
+			.find( commentID + '.depth-1' )
+			.first();
+		if ( ! comment.length ) {
 			return;
 		}
-
 		// Scroll to top level comment and adjust for admin bar.
-		var pos = comment.offset();
-		$( 'html,body' ).animate( {
-			scrollTop: pos.top - wpAdminBar
-		}, 1 );
+		const pos = comment.getBoundingClientRect();
+		window.scrollTo( {
+			top: window.scrollY + pos.top - wpAdminBar,
+			behavior: 'smooth',
+		} );
 	} );
 
-	// Scroll to comment if comment date link is clicked
-	$( '#comments' ).on( 'click', '.comment-date', function( e ) {
-		// Scroll to comment and adjust for admin bar
-		// Add 16px for child comments
-		var pos = $( this ).offset();
-		$( 'html,body' ).animate( {
-			scrollTop: pos.top - wpAdminBar - 16
-		}, 1 );
+	// Scroll to comment if comment date link is clicked.
+	document.querySelectorAll( '#comments .comment-date' ).forEach( ( element ) => {
```

Comment:
I don't believe this code path is even working anymore as I can't find the `#comments` parent. Do I have that wrong?

 Should we remove this smooth scrolling if that is the case?

Reply:
Yes, it was also not working here:
https://github.com/WordPress/wporg-developer/pull/532

Reply:
Fixed in https://github.com/WordPress/wporg-developer/pull/530/commits/9bb5d46f03c8d726519f9acedc9d298e09592c75 and https://github.com/WordPress/wporg-developer/pull/530/commits/2d7c1ed6c931a42b5c3deaf895204ef65db39b92